README.md

```diff
@@ -274,6 +274,28 @@ User.reindex
 Depending on how much data you have, reindexing may take a while but you only need to do it once.
 _ActiveStash will automatically index (and delete) data as it records are created, updated and deleted._
 
+## Uniqueness Validations
+
+Standard ActiveRecord uniqueness validations won't work when data is encrypted.
+While it *can* work in some cases when using deterministic encryption, we generally recommend against this approach
+(deterministic encryption is vulnerable to inference and chosen-plaintext attacks).
+
+Instead, you can include `ActiveStash::Validations` and your uniqueness validation will now work via a query
+to the CipherStash index on the validated field.
+If there is no index on the field then validations will fail.
+
+Also, note that as of now the `ActiveStash::Validations` module only supports a very simple uniqueness validation
+and doesn't yet support conditions or any option other than `:message`.
```

Comment:
Ah, that's going to be insufficient for [the initial use-case I'm after this feature for](https://github.com/heartcombo/devise/blob/v4.7.3/lib/devise/models/validatable.rb#L36).